controllers/apps/v2alpha1/generated.go

```diff
@@ -158,6 +169,17 @@ func generateListenerService(instance *appsv2alpha1.EMQX, listenerPorts []corev1
 		instance.Spec.ListenersServiceTemplate.Spec.Selector = instance.Spec.CoreTemplate.Labels
 	}
 
+	instanceAnnotations := instance.Annotations
+	if instanceAnnotations == nil {
+		instanceAnnotations = make(map[string]string)
+	}
+	delete(instanceAnnotations, "kubectl.kubernetes.io/last-applied-configuration")
+
+	serviceAnnotions := instance.Spec.ListenersServiceTemplate.Annotations
+	if serviceAnnotions == nil {
+		serviceAnnotions = make(map[string]string)
+	}
```

Comment:
`make(map[string]string)` can running in `mergeMap` function

Reply:
> `make(map[string]string)` can running in `mergeMap` function

have done it